.github/workflows/build.yml

```diff
@@ -61,6 +61,13 @@ jobs:
         env:
           NATIVE_BINARY_PATH: build/native/nativeCompile/migtool
 
+      - name: TestsNative reports
+        uses: actions/upload-artifact@v2
```

Comment:
worth starting to use `actions/upload-artifact@v4`

https://github.blog/2024-02-12-get-started-with-v4-of-github-actions-artifacts/

Reply:
I will open a separate PR to address this @alvaromartmart.